src/cli/cli.cpp

```diff
@@ -6820,75 +6823,147 @@ template <> otError Interpreter::Process<Cmd("targetpower")>(Arg aArgs[])
+        "extaddr",
+        "rloc16",
 #if OPENTHREAD_FTD
         "child table",
         "childip",
-        "neighbor table",
-        "router table",
-        "nexthop",
-        "eidcache",
 #if OPENTHREAD_CONFIG_MLE_DEVICE_PROPERTY_LEADER_WEIGHT_ENABLE
         "deviceprops",
 #endif
-#endif // OPENTHREAD_FTD
+        "eidcache",
+        "neighbor table",
+        "neighbor conntime",
+        "nexthop",
+        "router table",
+#endif
+
+    // Border Router info
+#if OPENTHREAD_CONFIG_BORDER_ROUTING_ENABLE
+        "br state",
+        "br omrprefix",
+        "br onlinkprefix",
+        "br prefixtable",
+#if OPENTHREAD_CONFIG_BORDER_ROUTING_MULTI_AIL_DETECTION_ENABLE
+        "br multiail",
+#endif
+#if OPENTHREAD_CONFIG_BORDER_ROUTING_TRACK_PEER_BR_INFO_ENABLE
+        "br peers",
+#endif
+        "br routers",
+#if OPENTHREAD_CONFIG_BORDER_ROUTING_DHCP6_PD_ENABLE
+        "br pd state",
+        "br pd omrprefix",
+#endif
+#endif
+
+    // Service info
+#if OPENTHREAD_CONFIG_BORDER_ROUTING_ENABLE
+        "br rdnsstable",
+#endif
+#if OPENTHREAD_CONFIG_NAT64_TRANSLATOR_ENABLE
+        "nat64 state",
+        "nat64 mappings",
+        "nat64 counters",
+#endif
 #if OPENTHREAD_CONFIG_SRP_CLIENT_ENABLE
         "srp client state",
+        "srp client server",
         "srp client host",
         "srp client service",
-        "srp client server",
 #endif
 #if OPENTHREAD_CONFIG_SRP_SERVER_ENABLE
         "srp server state",
         "srp server addrmode",
         "srp server host",
         "srp server service",
 #endif
+
+        // Radio & Link info
+        "ccathreshold",
+#if OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE || OPENTHREAD_CONFIG_MAC_CSL_TRANSMITTER_ENABLE
+        "csl",
+        "csl accuracy",
+        "csl uncertainty",
+#endif
+#if OPENTHREAD_CONFIG_PLATFORM_RADIO_COEX_ENABLE
+        "coex metrics",
+#endif
+#if OPENTHREAD_CONFIG_LINK_METRICS_MANAGER_ENABLE
+        "linkmetricsmgr show",
+#endif
+#if OPENTHREAD_CONFIG_MULTI_RADIO
+        "multiradio",
+        "multiradio neighbor list",
+#endif
 #if OPENTHREAD_CONFIG_RADIO_LINK_TREL_ENABLE
         "trel",
         "trel peers",
+        "trel counters",
 #endif
-#if OPENTHREAD_CONFIG_BORDER_ROUTING_ENABLE
-        "br state",
-        "br omrprefix",
-        "br onlinkprefix",
-        "br prefixtable",
-#if OPENTHREAD_CONFIG_NAT64_BORDER_ROUTING_ENABLE
-        "br nat64prefix",
+
+        // Counters & History
+        "counters ip",
+        "counters mac",
+        "counters mle",
+#if OPENTHREAD_CONFIG_IP6_BR_COUNTERS_ENABLE
+        "counters br",
 #endif
-#if OPENTHREAD_CONFIG_BORDER_ROUTING_DHCP6_PD_ENABLE
-        "br pd state",
-        "br pd omrprefix",
+#if OPENTHREAD_CONFIG_HISTORY_TRACKER_ENABLE
+        "history netinfo",
+        "history neighbor",
+        "history router",
+        "history prefix",
+        "history route",
 #endif
+#if OPENTHREAD_FTD && OPENTHREAD_CONFIG_MESH_DIAG_ENABLE
+        "meshdiag topology ip6-addrs children",
```

Comment:
I missed the addition of the `meshdiag` command to the `debug` list.

I'm not sure this is a good idea. The `debug` command is intended to dump all available state information quickly. Therefore, all commands in the `debug` list should retrieve information that is already stored on the device and not require any network interaction.

Reply:
sgtm to skip it here.